lib/cisco_node_utils/ace.rb

```diff
@@ -192,5 +232,99 @@ def dst_port
     def dst_port=(src_port)
       @set_args[:dst_port] = src_port
     end
+
+    def tcp_flags
+      match = ace_get
+      return nil if match.nil?
+      match.names.include?('tcp_flags') ? match[:tcp_flags] : nil
+    end
+
+    def tcp_flags=(tcp_flags)
+      @set_args[:tcp_flags] = tcp_flags
+    end
+
+    def established
+      match = ace_get
+      return nil if match.nil?
+      match.names.include?('established') ? true : false
+    end
+
+    def established=(established)
+      @set_args[:established] = established == true ? 'established' : ''
```

Comment:
The correction will set established when empty string is passed as argument, which we do not want.